Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
+        }
+
+        //[TestCase(true)]
```

Comment:
I guess the dynamic case will be enabled once we support stride in `DynamicVertexBuffer.SetData`?